src/main/java/inventorysetups/InventorySetupsPersistentDataManager.java

```diff
@@ -231,6 +276,28 @@ private void handleMigrationOfOldData()
 //				log.info("Removing old data key");
 //				configManager.unsetConfiguration(CONFIG_GROUP, CONFIG_KEY_SETUPS);
 //			}
+//		}

```

Comment:
I think we can enable this unset of the original V1 data (been over 1.5 years since that update it looks like). Additionally, line 259 to 266 can be turned into a `loadV1Setups` function

Reply:
I left out the last three lines to match the others (not running `updateConfig`, `inventorySetups.clear`, and `configManager.setConfiguration` in `loadV1Setups`.) Let me know if I misunderstood.

Also uncommented the V1-removing-code.